### PR TITLE
New exercise 2

### PR DESCRIPTION
**Pull request by aba-yom-mee**
Resubmission for Module 2 Project

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -1,41 +1,55 @@
 from flask import Flask, redirect, render_template, request, url_for
+import trello_cards
 import session_items as session
+import os
+from card import Card
```
Comment (Jonesey13) [inline app.py:4]: I get an error from this line when running locally.
* Did you forget to include a `card.py` in this review?

**Review by Jonesey13 — CHANGES_REQUESTED** [review]
Currently this app does not work for me due to a missing file `card.py`. 

Please could you commit and push this file so that I can continue the review?

File: app.py (hunk 1 above)
Comment (aba-yom-mee) [reply app.py:4]: It's push now.

File: session_items.py [hunk 2]
```diff
@@ -1,77 +1,47 @@
...
+        print(card ['name'], card['desc'])        
+    return todos
 
-    # Determine the ID for the item based on that of the previously added item
-    id = items[-1]['id'] + 1 if items else 0
 
-    item = { 'id': id, 'title': title, 'status': 'Not Started' }
+def get_done_items():
+    """ Simple attempt to get all done cards from Trello. """
+    response = requests.get(f"https://api.trello.com/1/lists/{os.getenv('DONE_idList')}/cards?key={os.getenv('TRELLO_API_KEY')}&token={os.getenv('TRELLO_API_TOKEN')}")
```
Comment (Jonesey13) [inline session_items.py]: You can pass the query parameters as a dictionary:
```suggestion
    params = { "key":  os.getenv('TRELLO_API_KEY'), "token": os.getenv('TRELLO_API_TOKEN')}
    response = requests.get(f"https://api.trello.com/1/lists/{os.getenv('DONE_idList')}/cards", params = params)
```

File: trello_cards.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import requests
```
Comment (Jonesey13) [inline trello_cards.py:1]: I'd be tempted to move the code from `session_items.py` into this file.

File: templates/index.html [hunk 4]
```diff
@@ -9,57 +9,35 @@ <h1 style="color: blue" class="display-4">To-Do App</h1>
...
             
             <div class="col-md-12"></div>
             <form action="/" method="POST">
-                    <h2>
-                        <label style="color: blue">Add New Item</label>
-                        <input class="form-control" id="name"  type="text" name="title">
-                        <button type="submit" class="btn btn-success">Create New Item</button>
+                    <h4>
+                        <label style="color: blue" for="name">Add Item</label>
+                        <p><input type="text" id="name" name="name"><input type="text" id="desc" name="desc"></p>
```
Comment (Jonesey13) [inline templates/index.html]: I'd split this over 2 lines and add placeholders:
```suggestion
                        <p>
                            <input type="text" id="name" name="name" placeholder="Title">
                            <input type="text" id="desc" name="desc" placeholder="Description">
                        </p>
```

**Review by Jonesey13 — APPROVED** [review]
Looks to be all working!

Left a few comments but feel free to merge this PR in whenever you are ready 👍

**Pull request merged**